js/main.js

```diff
@@ -1,2 +1,27 @@
 /* global data */
 /* exported data */
+
+var $imageDisplay = document.querySelector('#display-image');
+var $photoURL = document.querySelector('#image-input');
+
+$photoURL.addEventListener('input', function (e) {
+  $imageDisplay.src = $photoURL.value;
+  if ($photoURL.value === '') {
+    $imageDisplay.src = 'images/placeholder-image-square.jpg';
+  }
+});
+
+var $titleBox = document.querySelector('#input-box');
+var $notesBox = document.querySelector('textarea');
+var $form = document.querySelector('form');
+
+$form.addEventListener('submit', function (e) {
+  e.preventDefault();
```

Comment:
one this to add to this function is to reset the placeholder image after the form is submitted so that it matches the example.